electrum/tests/test_auxpow.py

```diff
@@ -37,61 +69,52 @@ def test_deserialize_auxpow_header_explicit_coinbase(self):
         self.assertEqual(expected_coinbase_txid, observed_coinbase_txid)
 
         coinbase_merkle_branch = header_auxpow['coinbase_merkle_branch']
-        self.assertEqual(5, len(coinbase_merkle_branch))
-        self.assertEqual('f8f27314022a5165ae122642babb28dd44191dd36f99dad80b4f16b75197dde0', coinbase_merkle_branch[0])
-        self.assertEqual('c8a9dc420e17dee7b04bc0174c7a37ed9e5bc3f0ea0fdfe0b5d24bfc19ecedb0', coinbase_merkle_branch[1])
-        self.assertEqual('0ce9c5b98e212527e4aa7b9298435dc4e8f4dfc4dc63b7c89c06300637c33620', coinbase_merkle_branch[2])
-        self.assertEqual('3b6d0c4122a5b047cb879a440461839f0446f6bd451f01c6f0b14b6624e84136', coinbase_merkle_branch[3])
-        self.assertEqual('458500be38a68b215112df5e52d9c08fdd52034fb2005ce15d2a42be28e436cb', coinbase_merkle_branch[4])
+        self.assertEqual(header_auxpow['coinbase_merkle_branch'], [
+            "f8f27314022a5165ae122642babb28dd44191dd36f99dad80b4f16b75197dde0",
+            "c8a9dc420e17dee7b04bc0174c7a37ed9e5bc3f0ea0fdfe0b5d24bfc19ecedb0",
+            "0ce9c5b98e212527e4aa7b9298435dc4e8f4dfc4dc63b7c89c06300637c33620",
+            "3b6d0c4122a5b047cb879a440461839f0446f6bd451f01c6f0b14b6624e84136",
+            "458500be38a68b215112df5e52d9c08fdd52034fb2005ce15d2a42be28e436cb",
+        ])
 
         coinbase_merkle_index = header_auxpow['coinbase_merkle_index']
         self.assertEqual(0, coinbase_merkle_index)
 
         chain_merkle_branch = header_auxpow['chain_merkle_branch']
```

Comment:
It looks like this PR removes all code that uses the `chain_merkle_branch` variable. If that's accurate, I think this variable should be removed as well since it's unused.

Reply:
Also removed this one.